tests/test_upload_mission.py

```diff
@@ -9,11 +9,21 @@
 import json
 from pathlib import Path
 
-from tests.fixtures import BASE_CONFIG, MODLIST_DICT, MODLIST_JSON
+import pytest
+
+from tests.fixtures import (
+    BASE_CONFIG,
+    MODLIST_DICT,
+    MODLIST_DICT_VERSIONLESS,
+    MODLIST_JSON,
+)
 from zeusops_bot.reforger_config_gen import ReforgerConfigGenerator, extract_mods
 
 
-def test_upload_edits_files(base_config: Path, mission_dir: Path):
+@pytest.mark.parametrize(
+    "strip_versions", (True, False), ids=["strip versions", "keep versions"]
```

Comment:
See below: don't need to change which mods value it is, if you can pass it as fixture/parameter
```suggestion
    "strip_versions,mods", [(True,MODLIST_JSON), (False, MODLIST_JSON_VERSIONLESS)], ids=["strip versions", "keep versions"]
```